### PR TITLE
🌱 Minor cleaning of M3DT manager code

### PR DESCRIPTION
**Pull request by lentzi90**
**What this PR does / why we need it**:

This improves naming, adds comments, etc. No functional changes!

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: controllers/metal3datatemplate_controller.go [hunk 1]
```diff
@@ -67,98 +67,98 @@ type Metal3DataTemplateReconciler struct {
...
-	if capm3DataTemplate.ObjectMeta.DeletionTimestamp.IsZero() {
+	if metal3DataTemplate.ObjectMeta.DeletionTimestamp.IsZero() {
 		if err != nil {
-			metadataLog.Info("Error fetching cluster. It might not exist yet, Requeuing")
+			log.Info("Error fetching cluster. It might not exist yet, Requeuing")
 			return ctrl.Result{}, nil
 		}
 	}
 
 	// Create a helper for managing the metadata object.
```
Comment (kashifest) [inline controllers/metal3datatemplate_controller.go:110]: may be the comment needs an update as well ?

**Review by kashifest — COMMENTED** [review]
/approve
Nice improvement @lentzi90

File: controllers/metal3datatemplate_controller.go (hunk 1 above)
Comment (lentzi90) [reply controllers/metal3datatemplate_controller.go]: Thanks, fixed!

**Review by Sunnatillo — COMMENTED** [review]
/approve

**Comment by lentzi90** [comment]
/override metal3-centos-e2e-integration-test-main
This test is broken at the moment and this PR does not make any functional changes anyway.

**Pull request merged**